cognito-idp/build.gradle.kts

```diff
@@ -102,9 +114,6 @@ kotlin {
                 implementation(kotlin("test-js"))
             }
         }
-        all {
-            languageSettings.enableLanguageFeature("InlineClasses")
```

Comment:
This should not be needed 🤔

Reply:
If it still compiles... indeed :dancers: :D